d3mola/Events-Manager
# #152816191 Setup sequelize and postgres

#### What does this PR do?
Sets up sequelize and postgres and connects to the database.

#### Description
The following actions were performed
- Installation of sequelize and sequelize-cli
- Creation of a sequelize config file `.sequelizerc`
- Addition of  database information to `server/config/config.json` file
- Installation of `dotenv` to keep sensitive data
- Creation of `user`, `event` and `center` models
- Generation of migration files for the above models
- Running migrations to persist models to the database

#### How should this be manually tested?
- clone the repo
- download and install [postgres](https://www.postgresql.org/) to your local machine
- install dependencies `npm install`
- run `node_modules/.bin/sequelize db:migrate`
- check postgres database to see the generated tables

#### What are the relevant pivotal tracker stories?
152816191
